DonjinKrawler-Client/src/main/java/donjinkrawler/GameMap.java

```diff
@@ -2,23 +2,89 @@
 
 import javax.swing.*;
 import java.awt.*;
+import java.awt.image.ImageObserver;
+
 
 public class GameMap {
 
-    private final int x = 0;
-    private final int y = 0;
+    private final int x = 250;
+    private final int y = 250;
     private Image image;
 
-    public GameMap() {
-        loadImage();
+    private int id;
+    private Door[] doors = new Door[4];
+
+    public GameMap(int id, int[] doorLocations) {
+        if (doorLocations[0] == 1)
+            doors[0] = new Door(Game.DoorDirection.LEFT,0,250);
```

Comment:
Visos absolute coordinates reiškia, kad turbūt nelabai galėsim resizint pačio žaidimo, correct? Turbūt out of scope šitam PR'ui, bet reikėtų paieškoti kaip galėtume turėt labiau flexible UI, arba bent padidinti iki pakenčiamos rezoliucijos